osxcollector/output_filters/shadowserver/lookup_hashes.py

```diff
@@ -39,9 +41,8 @@ def _should_add_threat_info_to_blob(self, blob, threat_info):
         Returns:
             boolean
         """
-        if blob.get('file_path', '').endswith(threat_info.get('filename', '')):
```

Comment:
Would there be a logic error here if, for example? :
blob.get('file_path', '') == "/path/long_file_name.txt"
threat_info.get('filename', '') == "e.txt"

^ The if statement would return true, even though the basename of the file_path does not match the filename from threat_info.